trulens_eval/examples/snowflake_demo/feedback.py

```diff
@@ -34,6 +34,15 @@
     )  # choose a different aggregation method if you wish
 )
 
+provider = Cortex(model_engine="snowflake-arctic")
```

Comment:
can we define this with a call to `get_provider()` to keep the Cortex object cached?

Reply:
done